### ISSUE TITLE
Infrared Question

### ISSUE THREAD

**Issue by josht000** [jcmello/MSTHT#1]
Hello, thanks for sharing your work. I've tested this on infrared imagery and the results are promising but I know you have another work dedicated just to that. Are you able to give some high level advice on how to translate this to your “Top-Hat Transform for Enhancement of Aerial Thermal Images” paper? 

Thanks in advanced.

**Comment by jcmello** [comment]
Hello, This proposal has already been developed in the paper called "[Entropy and contrast enhancement of infrared thermal images using the multiscale top-hat transform](https://www.mdpi.com/1099-4300/21/3/244#:~:text=(3)%2C%20244%3B-,https%3A//doi.org/10.3390/e21030244,-Received%3A%2029%20December)".

**Comment by josht000** [comment]
@jcmello Yes Professor, **that** paper. Are you able to please release that code in your github? 

Thanks,
Josh
